doc/Type/Backtrace.pod6

```diff
@@ -68,6 +68,69 @@ Defined as:
 
 Returns a list of L<Backtrace::Frame> objects for this backtrace.
 
+=head2 method summary
+
+Defined as:
+
+    method summary(Backtrace:D: --> Str:D)
+
+Returns a summary string representation of the backtrace, filtered
+by C<!.is-hidden && (.is-routine || !.is-setting)>.
+
+    sub inner { say Backtrace.new.summary }
+    sub outer { inner; }
+    outer;
+
+    #OUTPUT:
+    #perl6 test.p6
+    #  in method new at SETTING::src/core/Backtrace.pm6 line 85
+    #  in sub inner at test.p6 line 1
+    #  in sub outer at test.p6 line 2
+    #  in block <unit> at test.p6 line 3
+
+=head2 method concise
+
+Defined as:
+
+    method concise(Backtrace:D: --> Str:D)
+
+Returns a concise string representation of the backtrace, filtered
+by C<!.is-hidden && .is-routine && !.is-setting>.
+
+    sub inner { say Backtrace.new.concise }
+    sub outer { inner; }
+    outer;
+
+    #OUTPUT:
```

Comment:
This output can be done in the same way as previous method definition is suggested.